tool/tctl/common/resource_command.go

```diff
@@ -604,6 +605,20 @@ func (rc *ResourceCommand) createInstaller(ctx context.Context, client auth.Clie
 	return trace.Wrap(err)
 }
 
+func (rc *ResourceCommand) createNode(ctx context.Context, client auth.ClientI, raw services.UnknownResource) error {
+	server, err := services.UnmarshalServer(raw.Raw, "node")
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	if !rc.force {
+		return trace.AlreadyExists("nodes cannot be created, only upserted")
```

Comment:
As a novice, I'd read this and think I was supposed to run `tctl upsert node.yml`.  I'd mention `-f`/`--force` here.

Reply:
That's already displayed by `Create()` in response to an `AlreadyExists` - thus said error instead of something "more correct" like `BadParameter`.